WebDriverAgentTests/IntegrationTests/FBScrollingTests.m

```diff
@@ -50,6 +50,24 @@ - (void)testSimpleScroll
   FBAssertVisibleCell(@"10");
 }
 
+- (void)testScrollInDirection
```

Comment:
This is tested already. You should try testing scrolling by using `application` which will cover your case.